project/frontend/src/containers/CreateCompetition/CreateCompetition.js

```diff
@@ -24,7 +27,22 @@ const CreateCompetition = (props) => {
   const onParticipantDelete = (participant) =>
     setParticipantList(participantList.filter((v) => v !== participant?.email));
 
+  function checkValidDate() {
+    const start = new Date(startDate);
+    const end = new Date(endDate);
+    const now = new Date();
+    const today = new Date(now.getFullYear(), now.getMonth(), now.getDate());
+    if (end < start) setValidDate(-1);
```

Comment:
Is this formatting ok? I don't personally like it but idk what the style guide says

Reply:
will have a look

Reply:
yeah google doesn't like that either hahaha
![image](https://user-images.githubusercontent.com/75593542/107318188-0f412900-6b01-11eb-8e78-bfc524e63470.png)